src/main/java/de/cas/mse/exercise/SumDigPower.java

```diff
@@ -5,37 +5,41 @@
 
 public class SumDigPower {
 
-	public List<Long> sumDigPow(long a, long b) {
+	public List<Long> findAndPrintEurekaNumbers(long minNumber, long maxNumber) {
```

Comment:
Print is more How then why/what. This is a return method. So getEurekaNumbers is good enough